src/milia/api/io.cljs

```diff
@@ -46,7 +46,7 @@
   Authorization, X-CSRFToken and Cache-control headers where necessary"
   [& {:keys [get-crsftoken? must-revalidate? accept-header]}]
   (let [temp-token (:temp-token @*credentials*)]
-    (into {} [(when (and temp-token (is-not-null? temp-token))
+    (into {} [(when (and (not-empty temp-token) (is-not-null? temp-token))
```

Comment:
cool, did not know that, I see it's a non-boolean which explains the lack of suffix question mark